.daily.yml

```diff
@@ -0,0 +1,20 @@
+schedules:
```

Comment:
Can't set up the pipeline from a fork, annoyingly, so it'll have to get merged first.